### ISSUE TITLE
Create tool for computing/plotting one-to-all distances/costs

### ISSUE THREAD

**Issue by pedrocamargo** [AequilibraE/qaequilibrae#6]
Tool is interesting for computing "isochrones" and there is a demand from users.

**Comment by Art-Ev** [comment]
Was added with "Impendance Matrix" isn't it @pedrocamargo ? (if yes we can close the issue)

**Comment by pedrocamargo** [comment]
Not really.  We have a many to many and one to one, but not one to many or one to all, although that is pretty simple to implement.
